### PR TITLE
Solved Boolean algebra interactive element issue

### LINKED ISSUE #556: Boolean Algebra Interactive Element Not Working Properly
## Describe the bug

Regardless of the Dark/ Light Mode, the Interactive Element i.e. the switches and the Gates Drop-down do not appear and the place is blank. When the user clicks on the Blank space, the switch appears along with the drop-down and the other switch does not appears. 

## To Reproduce

1.  Go to 'https://learn.circuitverse.org/docs/binary-algebra/boolean-algebra.html'
2.  Scroll down to 'https://learn.circuitverse.org/docs/binary-algebra/boolean-algebra.html#click-on-gate-to-toggle-between-other-gates'
3.  See error

## Expected behavior

It should be visible completely when the page loads and should not remain blank.

## Screenshots

Looks Blank when the page the page loads:
Screenshot from 2021-06-25 17-37-02

On first click on the blank space:
Screenshot from 2021-06-25 17-43-18

On second click on the second blank space:
Screenshot from 2021-06-25 17-43-23

## Desktop (please complete the following information):

* OS: [Ubuntu 20.04.2]
* Browser [Firefox Web Browser]

## Additional context

This issue is reserved for Open Source Day participants.

### LINKED ISSUE #581: Not getting switch when selecting not gate
## Describe the bug
On selecting not gate the switch disappears 

## To Reproduce
image

1.  Go to 'Binary Algebra'
2.  Click on 'Boolean Algebra'
3.  Scroll down to 'Click on gate to toggle between other gates'
4.  See error

## Expected behavior
It should also have switch just like other gates have

image

## Desktop (please complete the following information):

OS = Windows 10
Browser = Chrome 

## Are you working on this issue ?

Yes , i am working on this issue , will raise a pr soon .

### PR DESCRIPTION
**Pull request by Palaksharma23**
Fixes #556 
Fixes #581

#### Changes done:
- [] Task 1 I have made switches visible on reload 
- []  Task 2 I have also solved the issue of not getting a switch while selecting not gate 

#### Screenshots

image

https://user-images.githubusercontent.com/87171452/146640179-a735b697-41bf-496e-a4d3-4071734ef455.mp4

#### ✅️ By submitting this PR, I have verified the following

- [ yes ] Checked to see if a similar PR has already been opened 🤔️
- [ yes ] Reviewed the contributing guidelines 🔍️
- [ yes  ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ yes ] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Palaksharma23** [comment]
@manjotsidhu can you please once review my pull request .

**Review by ayan-biswas0412 — APPROVED** [review]
Good work @Palaksharma23 🎉
@manjotsidhu can we merge this? 
it is working fine according to me.

**Comment by manjotsidhu** [comment]
@ayan-biswas0412 can you check #531 issue in your safari ?

**Comment by Palaksharma23** [comment]
@manjotsidhu no sir sorry it is just ref to #531 i will update my first comment.

**Comment by Palaksharma23** [comment]
Done sir :)

File: _includes/bool.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
 
 <div id="boolean">
-    <div id="0_bool" class="switch" onClick="toggle_switch(0);"></div>
+    <div id="0_bool" style="background-image: url(/assets/images/switch_on.png);" class="switch" onClick="toggle_switch(0);"></div>
```
Comment (manjotsidhu) [inline _includes/bool.html:3]: Avoid inline CSS and prefer already used css/id selectors in internal CSS.

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by Palaksharma23** [comment]
@manjotsidhu okay sir i will commit it asap :)

**Comment by Palaksharma23** [comment]
done sir :) @manjotsidhu

**Review by manjotsidhu — APPROVED** [review]
Good work 👍

**Pull request merged**